nodes/node.go

```diff
@@ -10,19 +10,18 @@ import (
 	"github.com/stampzilla/gozwave/database"
 	"github.com/stampzilla/gozwave/events"
 	"github.com/stampzilla/gozwave/functions"
-	"github.com/stampzilla/gozwave/serialapi"
+	"github.com/stampzilla/gozwave/interfaces"
 )
 
-func New(address int, connection *serialapi.Connection, eventQue chan interface{}) *Node {
+func New(address int) *Node {
```

Comment:
addr Address 

I'd make Address a type, probably.

Also, why "int" as the underlying type?